docs/source/posts/2024/2024-07-01-week-4-robin.rst

```diff
@@ -0,0 +1,74 @@
+Week 4: Pipeline Improvements and Taking The Bot Public!
```

Comment:
Hi @robinroy03!

Great Jobs, everything is good for me, except for a few small improvements to be added for the next few times. See below